### PR TITLE
Maraboupy docstrings in Google syntax

### PR DESCRIPTION
**Pull request by kjulian3**
This pr edits all of the docstrings for Maraboupy classes and functions in preparation for automatic documentation generation.

Most edits just reformat existing docstrings, but some were added, such as for Marabou.createOptions and MarabouCore.cpp. 

A subsequent pr will add the code necessary to make html pages generated by these docstrings.

### CODE REVIEW

**Comment by kjulian3** [comment]
I also edited the file headers to a python syntax. Is this ok to do, @guykatzz?

**Comment by guykatzz** [comment]
No problem about the style. These comments are generated by a script: tools/update-copyright.pl, which is supposed to be re-run in order to update these comments. We will need to adjust it so that it properly handles Python files (or create a separate script for the python files). This is not urgent.

File: maraboupy/Marabou.py [hunk 1]
```diff
@@ -121,10 +117,22 @@ def solve_query(ipq, filename="", verbose=True, options=None):
...
+    Args:
+        numWorkers (int, optional): Number of workers to use in DNC mode, defaults to 4
+        initialTimeout (int, optional): Initial timeout in seconds for DNC mode before dividing, defaults to 5 
+        initialDivides (int, optional): Initial divides in DNC mode, defaults to 0
+        onlineDivides (int, optional): Online divides in DNC mode, defaults to 2
+        timeoutInSeconds (int, optional): Timeout duration for Marabouin seconds, defaults to 0
+        timeoutFactor (float, optional): Timeout factor for DNC mode, defaults to 1.5
+        verbosity (int, optional): Verbosity level for Marabou, defaults to 2
+        dnc (bool, optional): If DNC mode should be used, defaults to False
+
```
Comment (anwu1219) [inline maraboupy/Marabou.py:133]: Could we change the description of initialDivides to "Number of times to perform the initial partitioning. This creates 2^(initialDivides) sub-problems, defaults to 0" 
onlineDivides to "Number of times to perform the online partitioning when a sub-query times out. This creates 2^(onlineDivides) sub-problems, defaults to 2" .

Just to highlight that this is how many times we bisect the problem, not the number of created sub-problems.

**Comment by anwu1219** [comment]
This is great to have. Thanks for doing this!

File: maraboupy/Marabou.py (hunk 1 above)
Comment (kjulian3) [reply maraboupy/Marabou.py:135]: Changes made, thanks for the suggestion.

**Pull request merged**